modules/moa-faq-installing.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/moa-faq.adoc
+
+
+[id="moa-faq-installing"]
```

Comment:
Every heading and subheading in a module should have a unique ID with a `_{context}` suffix. E.g.:

```
[id="moa-faq-installing_{context}"]
```